### PR TITLE
Fixing PHP Notices

### PR DESCRIPTION
**Pull request by nupplaphil**
Fixing https://github.com/friendica/friendica/issues/6918#issuecomment-491009954
Fixing https://github.com/friendica/friendica/issues/6918#issuecomment-491490533
Fixing https://github.com/friendica/friendica/issues/6918#issuecomment-491492826
Fixing parts of https://github.com/friendica/friendica/issues/6918#issuecomment-493515358

### CODE REVIEW

**Comment by nupplaphil** [comment]
puh .. ok ;-)

**Comment by nupplaphil** [comment]
oh.. wrong reword ... closing

File: mod/redir.php [hunk 1]
```diff
@@ -70,7 +70,10 @@ function redir_init(App $a) {
 				&& is_array($_SESSION['remote']))
 			{
 				foreach ($_SESSION['remote'] as $v) {
-					if ($v['uid'] == $_SESSION['visitor_visiting'] && $v['cid'] == $_SESSION['visitor_id']) {
+					if (!empty($v['uid']) && !empty($_SESSION['visitor_visiting']) &&
```
Comment (MrPetovan) [inline mod/redir.php]: Please use `Session::get('visitor_visiting')` and `Session::get('visitor_id')` instead. It checks for session key existence and returns a default value (or `null` by default) if it doesn't.

File: mod/redir.php [hunk 2]
```diff
@@ -70,10 +71,9 @@ function redir_init(App $a) {
 				&& is_array($_SESSION['remote']))
 			{
 				foreach ($_SESSION['remote'] as $v) {
-					if (!empty($v['uid']) && !empty($_SESSION['visitor_visiting']) &&
-					    !empty($v['cid']) && !empty($_SESSION['visitor_id']) &&
-					    $v['uid'] == $_SESSION['visitor_visiting'] &&
-					    $v['cid'] == $_SESSION['visitor_id']) {
+					if (!empty($v['uid']) && !empty($v['cid']) &&
+					    $v['uid'] === Session::get('visitor_visiting') &&
```
Comment (MrPetovan) [inline mod/redir.php]: I'm not sure a strict comparison is warranted. In theory it should all be strings so we don't really need to be that strict.

**Pull request merged**